references/detection/train_pytorch_DDP.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (C) 2021-2024, Mindee.
+
+# This program is licensed under the Apache License 2.0.
+# See LICENSE or go to <https://opensource.org/licenses/Apache-2.0> for full license details.
+
+import os
+
+os.environ["USE_TORCH"] = "1"
+
+import datetime
+import hashlib
+import logging
+import multiprocessing 
```

Comment:
`import multiprocessing as mp`

That's required for the worker count if not directly passed